system/Helpers/array_helper.php

```diff
@@ -26,7 +26,11 @@
 	 */
 	function dot_array_search(string $index, array $array)
 	{
-		$segments = explode('.', rtrim(rtrim($index, '* '), '.'));
+		$segments = preg_split('/(?<!\\\)\./', rtrim($index, '* '), 0, PREG_SPLIT_NO_EMPTY);
```

Comment:
Should this be four backslashes?

Reply:
It doesn't matter in this case. 4 and 3 backslashes in a string will be converted to 2 backslashes.

Reply:
Hmm., ok.